buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -1273,6 +1282,16 @@ void configureHttpWait(WaitForHttpResource wait) {
         }
     }
 
+    void configureForRun() {
```

Comment:
I think I'd prefer these to just be exposed as settings which the `RunTask` explicitly sets rather than coupling `ElasticsearchNode` to `RunTask` in this way.